attributioncode/validator.go

```diff
@@ -17,13 +17,14 @@ import (
 const maxUnescapedCodeLen = 1010
 
 var validAttributionKeys = map[string]bool{
-	"source":     true,
-	"medium":     true,
-	"campaign":   true,
-	"content":    true,
-	"experiment": true,
-	"variation":  true,
-	"ua":         true,
+	"source":         true,
+	"medium":         true,
+	"campaign":       true,
+	"content":        true,
+	"experiment":     true,
+	"installer_type": true,
```

Comment:
Is there a document specifying how this field is to be used?

Reply:
Document linked from this bug includes spec for this field. https://bugzilla.mozilla.org/show_bug.cgi?id=1598111